parm/archive/gefs_extracted_ocean.yaml.j2

```
@@ -0,0 +1,36 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + ".ocean.t" + cycle_HH + "z." %}
+
+gefs_ocean:
+    name: "GEFS_OCEAN"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs_ocean.tar"
+    required:
+#select mem%03d and ensstat files required
+{% set members = [] %}
+{% for mem_nm in range(0, NMEM_ENS + 1) %}
+    {% do members.append("mem" ~ '%03d' % mem_nm ) %}
+{% endfor %}
+
+{% set res = (OCNRES|string())[0] ~ "p" ~ (OCNRES|string())[-2:] %}
```

This may resolve the issue I explained above regarding the ocean product grid we will want to archive for the reforecast. 

```suggestion
{% if OCNRES == "025" %}
    {% set res = "1p00" %}                                                                                                                                                                    
{% else %}
    {% set res = (OCNRES|string())[0] ~ "p" ~ (OCNRES|string())[-2:] %}
{% endif %}
```